packages/n/neovim/abi_used_libs

```diff
@@ -4,8 +4,7 @@ libc.so.6
 libluajit-5.1.so.2
 libluv.so.1
 libm.so.6
-libmsgpack-c.so.2
 libtree-sitter.so.0.25
 libunibilium.so.4
+libutf8proc.so.3
 libuv.so.1
-libvterm.so.0
```

Comment:
As far as I can tell neovim was the last consumer of this lib so it should be deprecated.

Reply:
I did already make [an issue about that](https://github.com/getsolus/packages/issues/5305) :P